staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -66,6 +77,22 @@ if ($global:ContainerRuntime -eq "containerd") {
     $KubeletArgList += @("--container-runtime=remote", "--container-runtime-endpoint=npipe://./pipe/containerd-containerd")
 }
 
+# Update args to get both IPs from NIC using the default route 0.0.0.0/0 if IsDualStackEnabled
+# and the --node-ip argument is not present in the existing kubelet arg list.
+if (($global:KubeproxyFeatureGates -contains "IPv6DualStack=true") -and !(Test-NodeIPArgExists $KubeletArgList)) {
```

Comment:
could we move this to cse scripts to check whether it exists in `$Global:ClusterConfiguration.Kubernetes.Kubelet.ConfigArgs` before writing it into `c:\k\kubeclusterconfig.json` in Write-KubeClusterConfig?
With this change,
1. we can return WINDOWS_CSE_ERROR_DUAL_STACK_NO_EXACT_TWO_IPS if we cannot get two IPs
2. we can append ipAddrs to KubeletArgList if KubeletArgList does not contain them
3. we do not need to change kubeletstart.ps1 and cannot do anything if it fails to get two IPs.

Reply:
updated